src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,76 @@
+(function(transport, global, console) {
+    "use strict";
+    
+    function transportSend(args, type) {        
+        // To avoid Runtime.evaluate messages
+        if(args[0] !== "Runtime.evaluate") {
+            transport.send("bramble-console", args, type);
+        }
+    }
+    
+    function generateRouter(fname) {
+        var args = Array.from(arguments).slice();
```

Comment:
Let's DRY this code out (Don't Repeat Yourself):

```js
// Replace console methods with our own versions for integrated dev tools logging
["log", "warn", ...].forEach(function(type) {
  console[type] = function() {
    var args = Array.from(arguments).slice();
    transportSend(args, type);
  };
});
```

When you're basically copy/pasting the same code a dozen times, it's way safer to write code that generates the code, since it becomes boilerplate, and it's easy to introduce bugs with a mistake in one of the items.